e2e_tests/tests/requirements.txt

```diff
@@ -3,12 +3,10 @@ appdirs
 pytest>=6.0.1
 pytest-timeout
 pexpect
-torch==1.4
-torchvision==0.5.0
-tensorflow==1.15.4
-# TF 1.15.4 has requirement numpy<1.19.0,>=1.16.0
-# and numpy==1.19.2 throws numpy.ufunc has the wrong size error
```

Comment:
If you can remove these comments, can you just unpin numpy?  This which are pinned to very precise versions without any clear explanation why are a total pain in the but to figure out if it's safe to unpin later.